applications/sdp/gpio/CMakeLists.txt

```diff
@@ -9,6 +9,8 @@ cmake_minimum_required(VERSION 3.20.0)
 find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
 project(emulated_gpio)
 
+include(${ZEPHYR_NRF_MODULE_DIR}/cmake/sdp.cmake)
```

Comment:
when making `sdp_assembly_generate()` a proper function, then this can be moved to:
https://github.com/nrfconnect/sdk-nrf/blob/2fc583f30d262ac40df254277e78f234f43c9823/CMakeLists.txt#L28

and function can be used as a regular CMake function by the samples which needs the functionality.